tests/test_managers.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from marketing_blocks.models import MarketingBlock
+
+pytestmark = pytest.mark.django_db(transaction=True)
+
+
+class TestMarketingBlockManagers(object):
+
+    @staticmethod
+    def _mock_create_blocks():
```

Comment:
This is not mocking. When you mock, you mimic a behavior with a fake behavior (ie override a method or a class).
Here you are providing fixtures, ie fake content.
Pytest provides a extensive framework for fixtures:
https://docs.pytest.org/en/latest/fixture.html

If you want to go even further, you can use factories: 
https://pytest-factoryboy.readthedocs.io/en/latest/

Reply:
Fixed by 9deefec

I did  not use the `@pytest.mark.parametrize` decorator on this module. Perhaps it makes more sense in the `esidoc` module.